netbox/users/api/nested_serializers.py

```diff
@@ -23,11 +23,17 @@ class Meta:
 
 class NestedUserSerializer(WritableNestedSerializer):
     url = serializers.HyperlinkedIdentityField(view_name='users-api:user-detail')
+    display = serializers.SerializerMethodField(read_only=True)
 
     class Meta:
         model = User
         fields = ['id', 'url', 'display', 'username']
 
+    def get_display(self, obj):
+        if obj.first_name and obj.last_name:
+            return f"{obj.username} ({obj.first_name} {obj.last_name})"
```

Comment:
We can use `get_full_name()` on the User instance to return the first and/or last name if either is defined:

```suggestion
        if full_name := obj.get_full_name():
            return f"{obj.username} ({full_name})"
```

Reply:
`get_display()` should be defined on `UserSerializer` as well.